misc/openapi-ballerina/modules/ballerina-client-generator/src/main/java/org/ballerinalang/client/generator/GeneratorConstants.java

```diff
@@ -40,18 +40,18 @@ public String toString() {
 
     public static final String CLIENT_TEMPLATE_NAME = "client";
     public static final String OPEN_API_TEMPLATE_NAME = "skeleton";
-    public static final String SWAGGER_TEMPLATE_NAME = "skeleton";
+    public static final String OPENAPI_TEMPLATE_NAME = "skeleton";
 
     public static final String TEMPLATES_SUFFIX = ".mustache";
     public static final String TEMPLATES_DIR_PATH_KEY = "templates.dir.path";
     public static final String DEFAULT_TEMPLATE_DIR = "/templates";
     public static final String DEFAULT_CLIENT_DIR = DEFAULT_TEMPLATE_DIR + "/client";
     public static final String DEFAULT_OPEN_API_DIR = DEFAULT_TEMPLATE_DIR + "/oas3";
-    public static final String DEFAULT_SWAGGER_DIR = DEFAULT_TEMPLATE_DIR + "/swagger2";
+    public static final String DEFAULT_OPENAPI_DIR = DEFAULT_TEMPLATE_DIR + "/swagger2";
```

Comment:
we need to change the name of this dir... lets asses the impact

Reply:
This is a temp location which is only used in the OpenApi Client Generator to generate the temp OAS file. Hence IMO will be able to change the location name.